src/chatClient.h

```diff
@@ -195,6 +195,8 @@ class ChatRoom: public chatd::Listener, public DeleteTrackable
 
     bool hasChatHandler() const;
 
+    virtual unsigned long numMembers() const = 0;
```

Comment:
I think it should count your own user as participants, so `2` for 1on1 and `mPeers.size() + 1` for groupchats.

Reply:
Fixed